src/lib/registration/ssl_certificate.rb

```diff
@@ -28,6 +50,42 @@ def self.download(url, insecure: false)
       load(result)
     end
 
+    # Path to temporal CA certificates (to be used only in instsys)
+    TMP_CA_CERTS_DIR = "/var/lib/YaST2/ca-certificates".freeze
+
+    # Update instys CA certificates
+    #
+    # update-ca-certificates script cannot be used in inst-sys.
+    # See bsc#981428 and bsc#989787.
+    #
+    # @return [Boolean] true if update was successful; false otherwise.
+    #
+    # @see CA_CERTS_DIR
+    # @see TMP_CA_CERTS_DIR
+    def self.update_instsys_ca
+      # Update database
+      Yast::Execute.locally("trust", "extract", "--format=openssl-directory", "--filter=ca-anchors",
+        "--overwrite", TMP_CA_CERTS_DIR)
+
+      # Copy certificates/links
+      files = Dir[File.join(TMP_CA_CERTS_DIR, "*")]
+      targets = ["pem", "openssl"].map { |d| File.join(CA_CERTS_DIR, d) }
+      targets.each do |subdir|
+        FileUtils.mkdir_p(subdir) unless Dir.exist?(subdir)
+        files.each do |file|
+          # FileUtils.cp does not seem to allow copying the links without dereferencing them.
+          Yast::Execute.locally("cp", "--no-dereference", "--preserve=links", file, subdir)
+        end
+      end
+
+      # Cleanup
+      FileUtils.rm_rf(TMP_CA_CERTS_DIR)
+      true
+    rescue Cheetah::ExecutionFailed => e
```

Comment:
Then I guess that [this documentation](https://github.com/yast/yast-yast2/blob/master/library/system/src/lib/yast2/execute.rb#L56) is outdated, right?